libraries/atermpp/source/aterm_io_text.cpp

```diff
@@ -38,14 +38,14 @@ static void write_string_with_escape_symbols(const std::string& s, std::ostream&
   {
     // This function symbol needs quotes.
     os << "\"";
-    for(std::string::const_iterator i=s.begin(); i!=s.end(); ++i)
+    for (char i : s)
```

Comment:
Ik zou graag zien dat de dubbele punt tegen de i aan staat. Dit is typografisch anders dan elder wordt gedaan. Dus concreet for(char i: s).